src/EnergyPlus/SingleDuct.cc

```diff
@@ -1971,6 +1943,16 @@ namespace SingleDuct {
                                 sd_airterminal(SysNum).SysName);
         }
 
+        // common report variable for all single duct air terminals
+        for ( int sdIndex = 1; sdIndex <= NumSDAirTerminal; ++sdIndex) {
+            SetupOutputVariable("Zone Air Terminal Outdoor Air Volume Flow Rate",
+                OutputProcessor::Unit::m3_s,
+                sd_airterminal(sdIndex).OutdoorAirFlowRate,
+                "System",
+                "Average",
+                sd_airterminal(sdIndex).SysName );
+        }
```

Comment:
In the future, you can make this even simpler with a range based loop.

```
        for ( auto const & terminal : sd_airterminal) {
            SetupOutputVariable("Zone Air Terminal Outdoor Air Volume Flow Rate",
                OutputProcessor::Unit::m3_s,
                terminal.OutdoorAirFlowRate,
                "System",
                "Average",
                terminal.SysName );
        }
```

Just something to throw in your back pocket for later.